packages/terra-form-input/src/Input.module.scss

```diff
@@ -53,24 +53,24 @@
 
     &:focus {
       background-color: var(--terra-form-input-focus-background-color, #fff);
-      background-size: var(--terra-form-input-focus-keyboard-animation-size);
+      background-size: var(--terra-form-input-focus-keyboard-animation-background-size);
```

Comment:
nit: I'd drop the `keyboard-animation` from the variable name to be consistent with other variable names.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/d89bf406444dc93fa1237dae7fc382f0ec92b6b0#diff-462afbdb742ea35bac7504edbfd87d7dR56